docs/docs/policies/index.md

```diff
@@ -0,0 +1,66 @@
+# Getting Started with Policies
+
+Tracee's policies are how you describe what tracee should collect and what actions it should take.
```

Comment:
"Tracee's policies describe what **<here is a word missing e.g. information>** tracee should collect and what actions it should take **in response to an event**."

Feel free to not add my suggestions